### ISSUE TITLE
Question Regarding Chat Assistant - Workflow Orchestration Dialogue API: /chat-messages

### ISSUE THREAD

**Issue by hwb96** [langgenius/dify#7458]
### Self Checks

- [X] I have searched for existing issues [search for existing issues](https://github.com/langgenius/dify/issues), including closed ones.
- [X] I confirm that I am using English to submit report (我已阅读并同意 [Language Policy](https://github.com/langgenius/dify/issues/1542)).
- [X] [FOR CHINESE USERS] 请务必使用英文提交 Issue，否则会被关闭。谢谢！:）
- [X] Please do not modify this template :) and fill in all the required fields.

### Provide a description of requested docs changes

Description
I have a question regarding the /chat-messages API in the context of chat assistant and workflow orchestration.

Can the conversation_id be customized during the first request, or does it need to be left empty so that the system automatically generates it? After that, I would use the automatically generated conversation_id for subsequent multi-turn conversations.

In my first request, the conversation_id is empty:

json
复制代码
{
    "inputs": {},
    "query": "What is your name?",
    "response_mode": "streaming",
    "conversation_id": "",
    "user": "abc-123",
    "files": []
}
Then, in the second request, I use the conversation_id returned in the streaming response to continue the conversation:

json
复制代码
{
    "inputs": {},
    "query": "Repeat my previous question.",
    "response_mode": "streaming",
    "conversation_id": "4ad40f8f-d390-4395-ba08-110548e387cca",
    "user": "abc-123",
    "files": []
}
This allows me to proceed with subsequent turns normally.

However, if I try to use a custom conversation_id in the first request:

json
复制代码
{
    "inputs": {},
    "query": "What is your name?",
    "response_mode": "streaming",
    "conversation_id": "4ad40f8f-d390-4395-ba08-110548e387cca",
    "user": "abc-123",
    "files": []
}
I receive the following error:

json
复制代码
{
    "code": "invalid_param",
    "message": "4ad40f8f-d390-4395-ba08-110548e387cca is not a valid uuid.",
    "params": "conversation_id"
}
I would appreciate your help in clarifying this behavior.

**Comment by hwb96** [comment]
@dosu I used an online tool to generate a UUID: `6a1d3527-7b31-4aa6-bf37-b0c9a5a4f176`.

Then I sent a request with the following body:

```json
{
    "inputs": {},
    "query": "hello",
    "response_mode": "streaming",
    "conversation_id": "6a1d3527-7b31-4aa6-bf37-b0c9a5a4f176",
    "user": "abc-12",
    "files": []
}
```

I still received the following error:

```json
{
    "code": "not_found",
    "message": "Conversation Not Exists.",
    "status": 404
}
```

**Comment by hwb96** [comment]
You previously said, "Yes, the conversation_id can be customized during the first request to the /chat-messages API." But now you're saying, "When creating a new conversation, you should omit the conversation_id in the first request. The system will automatically generate a new conversation_id for you." Which one should I trust? I'm even more confused now, and I really need a human to help me solve this issue. Thank you so much!

**Comment by leslie2046** [comment]
Yes, as you mentioned before, conversation_id  should be left empty in the first request , and  should fill it with the conversion_id returned by the first request for continue requests.
